src/Native/Runtime/Full/CMakeLists.txt

```diff
@@ -17,9 +17,11 @@ set(ASM_OFFSETS_CSPP ${RUNTIME_DIR}/../../Runtime.Base/src/AsmOffsets.cspp)
 if(WIN32)
     set(COMPILER_LANGUAGE "")
     set(PREPROCESSOR_FLAGS -EP)
+    set(ASM_OFFSETS_CPP ${RUNTIME_DIR}/AsmOffsets.cpp)
```

Comment:
Could you move the AsmOffsets.cpp to the windows subfolder since it is windows only now?